addons/api/tests/unit/serializers/credential-test.js

```diff
@@ -33,6 +33,34 @@ module('Unit | Serializer | credential', function (hooks) {
     });
   });
 
+  test('it serializes username_password type correctly on create no password change', function (assert) {
```

Comment:
Can we rephrase this to improve readability?

Reply:
Yes. I will do that now.